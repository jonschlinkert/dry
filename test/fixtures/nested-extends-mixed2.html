{% extends "nested-extends-mixed2" %}

{% block prepend="one" %}
one: "nested-extends-mixed2.html"
{% endblock %}

{% block append="two" %}
two: "nested-extends-mixed2.html"
{% endblock %}

{% block append="five" %}
five: "nested-extends-mixed2.html"
{% endblock %}

{% block "six" %}
six: "nested-extends-mixed2.html"
{% endblock %}

{% block prepend="seven" %}
seven: "nested-extends-mixed2.html"
{% endblock %}