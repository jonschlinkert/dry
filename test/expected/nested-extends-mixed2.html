
one: "nested-extends-mixed2.html"
one: "blocks/parent-mixed2.html"

two: "blocks/nested-extends-mixed2.html"
two: "blocks/parent-mixed2.html"
two: "nested-extends-mixed2.html"

three: "blocks/nested-extends-mixed2.html"
three: "blocks/parent-mixed2.html"

four: "blocks/nested-extends-mixed2.html"
four: "blocks/parent-mixed2.html"

five: "blocks/nested-extends-mixed2.html"
five: "nested-extends-mixed2.html"

six: "nested-extends-mixed2.html"

seven: "nested-extends-mixed2.html"
seven: "blocks/nested-extends-mixed2.html"
seven: "blocks/parent-mixed2.html"